pycloudlib/instance.py

```diff
@@ -33,6 +33,8 @@ def __init__(self, key_pair):
         self.port = '22'
         self.username = 'ubuntu'
 
+        self.connect_timeout = connect_timeout if connect_timeout else 60
```

Comment:
This prohibits setting a `connect_timeout` of 0; 0 is False-ish, so it will be set to 60 here.

```suggestion
        self.connect_timeout = connect_timeout if connect_timeout is not None else 60
```